app/models/offline_transaction_refund.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+
+# License: AGPL-3.0-or-later WITH WTO-AP-3.0-or-later
+# Full license explanation at https://github.com/houdiniproject/houdini/blob/master/LICENSE
+class OfflineTransactionRefund < ApplicationRecord
+	include Model::SubtransactionPaymentable
+	belongs_to :payment
+
+	delegate :gross_amount, :net_amount, :fee_total, :nonprofit, :supporter, to: :payment
```

Comment:
With the code as it is, I can't access `nonprofit` and `supporter` without delegating to payment.